src/Domain/Creators/BinaryCompactObject.cpp

```diff
@@ -258,19 +258,24 @@ BinaryCompactObject::BinaryCompactObject(
     // the spherical shell (for logarithmic or inverse mapping).
     const size_t addition_to_outer_layer_radial_refinement_level =
         initial_refinement_[44][2] - initial_refinement_[34][2];
-    const double radial_divisions_in_outer_layers =
+    const size_t radial_divisions_in_outer_layers =
         pow(2, addition_to_outer_layer_radial_refinement_level) + 1;
-    const double singularity_pos = 0.;
+    // Use the `Interval` class to divide the interval between
+    // `radius_enveloping_cube_` and `outer_radius_` into
+    // `radial_divisions_in_outer_layers` pieces. Choose
+    // `radius_enveloping_sphere_` as the first of those pieces.
     radius_enveloping_sphere_ =
         domain::CoordinateMaps::Interval{// Source interval
-                                         0., radial_divisions_in_outer_layers,
+                                         0., 1.,
                                          // Target interval
                                          radius_enveloping_cube_, outer_radius_,
                                          // Distribution in target interval
                                          radial_distribution_outer_shell_,
-                                         singularity_pos}(
-            // Inner radius in source interval
-            std::array<double, 1>{{1.}})[0];
+                                         // Position of the singularity for log
+                                         // and 1/r maps (in target interval)
+                                         0.}(std::array<double, 1>{
+            {// Inner radius in source interval
```

Comment:
Maybe add `that is mapped to target interval` to make clear that the result of the Interval map gives a result in the target interval (that `radius_enveloping_sphere_` is in the target interval) (and not that `Inner radius in source interval` is the result which `radius_enveloping_sphere_` is then set to)

The rest is clearer to me! Thank you :)